skimage/draw/tests/test_draw3d.py

```diff
@@ -1,23 +1,23 @@
 import numpy as np
 from numpy.testing import assert_array_equal, assert_allclose
-from nose.tools import raises
+from numpy.testing import assert_raises
 
 from skimage.draw import ellipsoid, ellipsoid_stats
 
 
-@raises(ValueError)
 def test_ellipsoid_sign_parameters1():
-    ellipsoid(-1, 2, 2)
+    with assert_raises(ValueError):
+        ellipsoid(-1, 2, 2)
 
 
-@raises(ValueError)
 def test_ellipsoid_sign_parameters2():
-    ellipsoid(0, 2, 2)
+    with assert_raises(ValueError):
```

Comment:
well… currently this fails, as numpy.testing.assert_raises relies on nose. I am currently fixing this.